.gitignore

```diff
@@ -58,4 +58,5 @@ typings/
 .env
 
 dist
-publish_dist
\ No newline at end of file
+publish_dist
+package-lock.json
```

Comment:
Im ignoring this since we are using Yarn for the project, just be careful locally to use yarn, otherwise you might end up with different versions